german/agot_artifacts/agot_artifacts_l_german.yml

```diff
@@ -49,13 +49,13 @@
  vs_darktalon_name:0 "Dunkelklaue"
  vs_darktalon_description:0 "Die Vyrwells von Dunkeltal schwingen dieses Langschwert seit Generationen. Auf dem versilberten Griff ist ein grimmiger Wyvern mit ausgebreiteten Flügeln eingraviert."
  vs_dawn_name:0 "Dämmerung"
- vs_dawn_description:0 "Dämmerung ist das sagenumwobene Schwert des Hauses Dayn. Das riesige Großschwert hat eine Klinge, so blass wie Milchglas, und einen goldenen Stern, der den Griff ziert. Die Legende besagt, dass ein gefallener Stern vom Gründer des Hauses Dayn bis zur Mündung des Torrentine verfolgt wurde, an dem Ort, der später Sternfall genannt werden sollte. Dämmerung wurde aus dem Herzen dieses Sterns geschmiedet und wird seither von den würdigsten Mitgliedern des Hauses Dayn geführt."
+ vs_dawn_description:0 "Dämmerung ist das sagenumwobene Schwert des Hauses Dayn. Das riesige Großschwert hat eine Klinge, so blass wie Milchglas, und einen goldenen Stern, der den Griff ziert. Die Legende besagt, dass ein gefallener Stern vom Gründer des Hauses Dayn bis zur Mündung des Torentin verfolgt wurde, an dem Ort, der später Sternfall genannt werden sollte. Dämmerung wurde aus dem Herzen dieses Sterns geschmiedet und wird seither von den würdigsten Mitgliedern des Hauses Dayn geführt."
  vs_defiance_name:0 "Trotz"
  vs_defiance_description:0 "Trotz ist ein Schwert mit Bronzegriff aus dem Besitz des Hauses Eisenschmied. Das Haus Eisenschmied erlangte über Jahrhunderte hinweg Bekanntheit als Lieferant von Hufeisen für die Kavallerie des Nordens. Diese Bekanntheit ermöglichte es ihnen schließlich, Trotz in Auftrag zu geben und ihre Stellung unter dem Adel des Nordens zu festigen."
  vs_disputant_name:0 "Streitlust"
  vs_disputant_description:0 "Streitlust ist ein kurzes Schwert mit dunkelgoldenem, fledermausförmigem Griff. Die Klinge ist schwarz wie die Nacht."
  vs_the_dagger_name:0 "Dolch mit Drachenbeingriff"
- vs_the_dagger_description:0 "Dieser Dolch ist entlang des gebogenen Griffs aus Drachenbein mit Rubinen besetzt. Die ungewöhnliche, halbgekrümmte Form und die Verwendung von Drachenknochen im Griff dieses verruchten Dolches zeigen, dass dieser Dolch im alten Valyria für eine extrem reiche Adelsfamilie geschmiedet wurde."
+ vs_the_dagger_description:0 "Dieser Dolch ist entlang des gebogenen Griffs aus Drachenknochen mit Rubinen besetzt. Die ungewöhnliche, halbgekrümmte Form und die Verwendung von Drachenknochen im Griff dieses verruchten Dolches zeigen, dass dieser Dolch im alten Valyria für eine extrem reiche Adelsfamilie geschmiedet wurde."

```

Comment:
Würde in der Beschreibung eher 2-mal Drachenknochen verwenden aber das im Namen beibehalten.